docs/wip_glassbox/entity-extension.md

```diff
@@ -112,9 +122,21 @@ DiscriminatorMap must always contain descendants' FQN because LoadORMMetadataSub
         Shopsys\FrameworkBundle\Model\Product\ProductFactoryInterface:
           alias: DreamProject\Model\Product\ProductFactory
       ```
+* Create a new data object in your `src/Shopsys/ShopBundle/Model` directory that extends already existing framework entity data
+* Create a factory for this entity data
+* Implement the factory interface from the framework
+    * eg. `class ProductDataFactory implements ProductDataFactoryInterface`
+  * Rewrite symfony configuration for the interface to alias your factory
+    * eg.
+      ```php
+        Shopsys\FrameworkBundle\Model\Product\ProductDataFactoryInterface:
+          alias: DreamProject\Model\Product\ProductDataFactory
+      ```
 * Now your extended entity is automatically used instead of the parent entity:
   * in hydrated Doctrine references
   * in the EntityManager, Repositories and QueryBuilders
   * in newly created entities
+  * in data objects
+    * *but still not in data objects created by forms*
```

Comment:
could you be more descriptive here please? What does it mean?

Reply:
This is a horrible mistake - this part is about entities. Removed